pkg/tsdb/loki/loki.go

```diff
@@ -95,6 +97,7 @@ func newInstanceSettings(httpClientProvider httpclient.Provider) datasource.Inst
 			return nil, err
 		}
 
+		opts.Middlewares = append(sdkhttpclient.DefaultMiddlewares(), errorsource.Middleware("errorsource"))
```

Comment:
@scottlepp just fyi, I decided to go with this instead of using `errorsource http client` because we have healtcheck tests that add `RoundTripper` to `httpClientProvider` to mock response. It would be more work to  rewrite tests than using this - also considering that errorsource http client is currently experimental.